app/build.gradle

```diff
@@ -114,22 +114,15 @@ configurations {
 dependencies {
     implementation 'io.sentry:sentry-android:1.7.9'
 
-    implementation "com.android.support:appcompat-v7:$support_libraries_version"
-    implementation "com.android.support:support-v4:$support_libraries_version"
-    implementation "com.android.support:cardview-v7:$support_libraries_version"
-    implementation "com.android.support:customtabs:$support_libraries_version"
-    implementation "com.android.support:design:$support_libraries_version"
-    implementation "com.android.support:leanback-v17:$support_libraries_version"
-    implementation "com.android.support:recyclerview-v7:$support_libraries_version"
-    implementation "com.android.support.constraint:constraint-layout:1.1.3"
-    implementation 'com.squareup.picasso:picasso:2.71828', {
-        // We temporarily don't include the packaged the support libs to avoid version mismatch.
-        // This can cause runtime errors so we should remove this ASAP! #883?
-        exclude group: 'com.android.support', module: 'exifinterface'
-        exclude group: 'com.android.support', module: 'support-annotations'
-        exclude group: 'com.android.support', module: 'support-compat'
-        exclude group: 'com.android.support', module: 'support-fragment'
-    }
+    implementation "androidx.appcompat:appcompat:$support_libraries_version"
+    implementation "androidx.legacy:legacy-support-v4:$support_libraries_version"
+    implementation "androidx.cardview:cardview:$support_libraries_version"
+    implementation "androidx.browser:browser:$support_libraries_version"
+    implementation "com.google.android.material:material:$support_libraries_version"
```

Comment:
This looks like it's new, do we need this?

Reply:
This is the new name for `com.android.support:design`.  Crazy, right?